XeonE3/centos-7/analytics/gst/Dockerfile.m4

```diff
@@ -1,9 +1,7 @@
-
-FROM centos:centos7 AS build
-WORKDIR /home
-define(`BUILD_LINKAGE',shared)dnl
-
-include(build-tools.m4)
+include(begin.m4)
+include(centos-repo.m4)
+include(cmake.m4)
+include(git.m4)
```

Comment:
What issue to be fixed by including git.m4

Reply:
New 'meson' based builds in GST world, some of them calls to 'git -C'. '-C' options was later introduced in git. And git that comes through yum in C7, is older and would not support that '-C', to make this work, we need to build newer git from source.

Reply:
I don't see this issue during the OWT centos build. It also includes gst.

Reply:
Let me check which exact 'gst' component needed it.

Reply:
Removed custom git, and not facing this anymore.

**Long story**: For some reason I had included custom built meson.m4 in C7 images, which installs Meson 0.53.1 as opposed to C7 yum install meson, which installs 0.55.1. This older meson gave problem with git, hence I wrote git.m4.

 But referring to @xwu2git 's owt recipe, I realized we dont need to source built meson for C7, after removing meson.m4 from those images, I didn't see git errors. Hence, removed git.m4.